task10-vue/src/components/SynopsisSection.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div :class="`op-characters ${readAll ? 'read-all' : ''}`">
```

Comment:
template element should have exactly 1 direct child, here you have a div and an article as direct children